eclipse-theia/theia-blueprint
# Update welcome page

#### What it does

Fixes: https://github.com/eclipse-theia/theia-blueprint/issues/280

This PR updates Theia Blueprint's welcome page to use the latest changes from Theia 1.41 leading to a reduction in lines of code. It also fixes two small typos that were present in the welcome page.

This update allows Blueprint to benefit from these changes:
https://github.com/eclipse-theia/theia/pull/12813
https://github.com/eclipse-theia/theia/pull/12750


#### How to test


1.  Open Theia application.
2.  Welcome page should use the latest changes from 1.41.

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
